ruby-challenges.rb

```diff
@@ -18,13 +30,50 @@
 
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
+def num(array)
+    array.inject(0, :+)
+end
+p num(nums_array1)
+p num(nums_array2)
 
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
+class Bike 
+    def initialize(model)
+        @model = model
+        @wheels = 2
+        @current_speed = 0
+    end
+    def bike_info
+        if @current_speed > 0
```

Comment:
Great work!  If you want to dig a little deeper into some shorter syntax, take a look at using ternary operators to replace your traditional if/else statements.